src/client/app/actions/barReadings.ts

```diff
@@ -91,7 +90,7 @@ function fetchMeterBarReadings(meterIDs: number[], timeInterval: TimeInterval):
 	return async (dispatch: Dispatch, getState: GetState) => {
 		const barDuration = getState().graph.barDuration;
 		dispatch(requestMeterBarReadings(meterIDs, timeInterval, barDuration));
-		const readings = await metersApi.barReadings(meterIDs, timeInterval, barDuration);
+		const readings = await compressedReadingsApi.meterBarReadings(meterIDs, timeInterval, Math.round(barDuration.asDays()));
```

Comment:
Since we have the API object model, we could probably do the toDays and rounding within the meterBarReadings if it gets a Duration.

Reply:
Yeah, that makes sense to me.